reformcloud-library/src/main/java/systems/reformcloud/meta/server/versions/SpigotVersions.java

```diff
@@ -160,9 +160,7 @@
     /**
      * ShortSpigot Versions
      */
-    SHORTSPIGOT_1_12_2("ShortSpigot 1.12.2", "1.12.2", 
-         "https://dl.shortspigot.sh/file/latest");
-    /**    
+    SHORTSPIGOT_1_12_2("ShortSpigot 1.12.2", "1.12.2", "https://dl.shortspigot.sh/file/latest");   
```

Comment:
You sould REMOVE ShortSpigot because of non-working download servers... Why do you add it here?

Reply:
Because I don't want errors when I edit it. But I am already Removing it.

Reply:
You have to remove all usages before pushing 🤷‍♂

Reply:
I know. :D I have acutally some Problems in the CloudServerStartup Class but I will fix this.

Reply:
So its now Finished.